wiki/Disambiguation/Mod/de.md

```diff
@@ -0,0 +1,7 @@
+# Mod
+
+**Mod**kann folgendes bedeuten:
+
+-   Ein Moderator, wie z.B. ein [QAT-](/wiki/QAT) oder [GMT-](/wiki/GMT)Mitglied.
+-   Eine Person die über eine Beatmap drüber guckt und ihr Feedback zu dieser abgibt.
```

Comment:
- `Person, die`
- "drübergucken" ist seeehr umgangssprachlich. Würde ich vllt ersetzen.

Reply:
Eine Person, die Beatmaps prüft und Feedback zu diesen gibt?

Reply:
Prüfen klingt allerdings so formal wie als wenn es eine Aufgabe ist die die Person erledigen muss.

Reply:
Hm... Kontrollieren vllt? Da fehlt ja eigentlich nur ein gutes Synonym.. vllt mal in nem Thesaurus checken. Ich bin mir da auch nicht so 100%ig sicher

Reply:
"Eine Person die Beatmaps rezensiert/bewertet"?